itou/scripts/management/commands/update_suspensions_end_at.py

```diff
@@ -0,0 +1,121 @@
+import io
+from datetime import datetime as dt
+
+import pandas
+from dateutil.relativedelta import relativedelta
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from itou.approvals.models import Approval, Suspension
+
+
+FIRST_SCRIPT_RUNNING_DATE = timezone.make_aware(dt(2023, 2, 1, 17, 44))
```

Comment:
Plutôt que d’utiliser de la logique pour gérer les TZ, je recommanderai de construire directement un objet `datetime` avec `tzinfo` :

```suggestion
FIRST_SCRIPT_RUNNING_DATE = datetime.datetime(2023, 2, 1, 16, 44, tzinfo=datetime.timezone.utc)
```